content/barbuddies/vanadie_dwh.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Vanadie
+pronouns: 
+  nl: die/diens
+  en: they/them
+sites:
+  - dwh
+nl:
+  "Heyhey, leuk dat je eens langs wil komen bij DWH! Zelf heb ik hier tegen het eind van 2021 voor het eerst voet 
```

Comment:
kunnen we dit aanpassen naar "DWH"->"onze vereniging" (of bar oid)?

dan hoeven we niet twee versies te maintainen